applications/dairdrop.md

```diff
@@ -0,0 +1,294 @@
+- **Full-Time Equivalent (FTE):**  5,0
+<!-- - Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE) -->
+- **Total Costs:** 24,000 
+- **DOT %:** 50%
+> Note: the funds from this grant will not be used for airdrops to users but will be allocated to the development of the application.
+
+### Milestone 1 — On-chain pallets and Ink contracts, airdrop indexer.
+
+- **Estimated duration:** 1.5 month
+- **FTE:**  2,5
+- **Costs:** 12,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one.
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0  |
+| **0b.** | Documentation | Create detailed documentation for each module. |
+| **0c.** | Testing and Testing Guide | Over 70% code test coverage. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 1. | Substrate module: dairdorp | Provide an airdrop strategy based on airdrop files and Merkle tree algorithms. |
+| 2. | Smart contracts:  dairdorp-ink| Provide an airdrop strategy based on airdrop files and Merkle tree algorithms. |
+| 3. | Smart contracts: dairdrop-vrf| Provide an airdrop strategy based on the VRF algorithm.|
+| 4. | Substrate module: dairdorp-vrf | Provide an airdrop strategy based on the VRF algorithm. |
+| 5. | indexer| An on-chain airdrop activity indexer that monitors on-chain airdrop status timely. |
+| 6. | Substrate solo chain | Dota 2.0 test chain used for testing airdrop functionality and preparing for the launch of the Dota 2.0 mainnet.  (Before the JAM protocol is fully developed, we hope to create an solo chain. Once we gain more support from developers and users, we will gradually transition to a parachain that shares security with Polkadot’s JAM)|
+
+
+
+### Milestone 2 — UI/UX design, web server (backend), and front-end development.
+
+- **Estimated Duration:** 1.5 month
+- **FTE:**  2.5
+- **Costs:** 12,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | Apache 2.0  |
+| **0b.** | Documentation | Provide documentation for the app. |
+| **0c.** | Testing and Testing Guide | Provide manual test cases for each feature and achieve at least 70% test coverage for each API in the web service. |
+| **0d.** | Docker | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an **article** that explains how it offers more convenient and simpler services in the airdrop space, how it differs from traditional airdrops, who should use it, and how to use it.  |
+| 1. | UI/UX design | Design user-friendly interactive pages to enhance the user experience. |
```

Comment:
We don't fund the design phase of a project. Given that this is the last milestone, it also doesn't make sense as we would only see the design when it's already implemented. Please remove.